deploy

```diff
@@ -98,6 +98,9 @@ def parse_args():
                         help="Repo and ref to use for FCOS config")
     parser.add_argument("--bucket", metavar='BUCKET',
                         help="AWS S3 bucket to use")
+    parser.add_argument("--kvm-selector", help="KVM selector",
```

Comment:
Can we document this in the `HACKING.md` doc where we describe the different switches? Also, I think we should drop `required` and default to what the official pipeline needs for now, which would be `legacy-oci-kvm-hook`. That wait, `./deploy --update --official` still works the same.